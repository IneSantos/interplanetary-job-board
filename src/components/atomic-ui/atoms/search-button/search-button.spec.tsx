import React from "react";
import { render } from '@testing-library/react';
import SearchButton from "./search-button";

describe("SearchButton atom tests", () => {
  test(`SearchButton component render`, () => {
    const { container } = render(
      <SearchButton/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
