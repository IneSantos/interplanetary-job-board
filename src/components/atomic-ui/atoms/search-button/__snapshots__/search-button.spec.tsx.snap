// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchButton atom tests SearchButton component render 1`] = `
<button
  class="search-button"
  type="submit"
>
  <i
    class="fas fa-search"
  />
</button>
`;
